deeplake/integrations/mmdet/mmdet_.py

```diff
@@ -1324,9 +1324,9 @@ def _train_detector(
     # register eval hooks
     if validate:
         val_dataloader_default_args = dict(
-            samples_per_gpu=1,
-            workers_per_gpu=1,
-            dist=False,
+            samples_per_gpu=len(cfg.gpu_ids),
```

Comment:
this should be batch_size, not len(cfg.gpu_ids)

Reply:
yes, this is a typo